### DISCUSSION TITLE
Xcode 13 : KingfisherSwiftUI Task failed with exit code 65: during Carthage dependency installation

### DISCUSSION THREAD

**Discussion by Sumeet-Mediacorp** [onevcat/Kingfisher#1813] [Q&A]
I am facing below error after migrating to Xcode 13, though the issue doesn't come with Xcode 12.5: 

**Terminal Logs:**

`*** Building scheme "KingfisherSwiftUI" in Kingfisher.xcworkspace
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -workspace /Users/admin/Documents/myApp_iOS/Carthage/Checkouts/Kingfisher/Kingfisher.xcworkspace -scheme KingfisherSwiftUI -configuration Release -derivedDataPath /Users/admin/Library/Caches/org.carthage.CarthageKit/DerivedData/13.0_13A233/Kingfisher/5.15.6 -sdk iphoneos ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive -archivePath /var/folders/ry/j50lnscs7p11r80mr5m82_8h0000gn/T/Kingfisher SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO STRIP_INSTALLED_PRODUCT=NO (launched in /Users/admin/Documents/myApp_iOS/Carthage/Checkouts/Kingfisher)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/ry/j50lnscs7p11r80mr5m82_8h0000gn/T/carthage-xcodebuild.sHIafd.log
admin@Admins-MacBook-Pro myApp_iOS % 
`
Did anyone else faced the same issue? 

**carthage-xcodebuild.sHIafd.log logs** : 

** ARCHIVE FAILED **

The following build commands failed:
	CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler (in target 'KingfisherSwiftUI' from project 'Kingfisher')
	CompileSwiftSources normal armv7 com.apple.xcode.tools.swift.compiler (in target 'KingfisherSwiftUI' from project 'Kingfisher')
	CompileSwift normal armv7 (in target 'KingfisherSwiftUI' from project 'Kingfisher')
(3 failures)

**Comment by onevcat** [comment]
Looks like it is the same as #1802. If you are still using Kingfisher v6, it is a Xcode 13 known issue and now there is no workaround to support iOS 11 or below. Kingfisher v7 update the support target to iOS 12 and should compile well in Xcode 13.

If you still need to support iOS 10 and 11, keep using Xcode 12.5 for a while and see if Apple can fix this issue in Xcode 13.1.
